installers-conda/resources/post-install.sh

```diff
@@ -109,34 +120,35 @@ fi
 if [[ $OSTYPE = "linux"* ]]; then
     cat <<EOF
 
+###############################################################################
+#                             !!! IMPORTANT !!!
 ###############################################################################
 Spyder can be launched by standard methods in Gnome and KDE desktop
-environments. Additionally, Spyder can be launched in Gtk-based desktop
-environments (e.g. Xfce) from the command line:
-
-$ gtk-launch spyder
-
-Spyder can also be launched from the command line for all Linux variants by:
+environments. It can also be launched from the command line on all Linux
+distros with the command:
 
 $ spyder
 
 To uninstall Spyder, run the following from the command line:
 
 $ uninstall-spyder
 
-#####################
-# !!! IMPORTANT !!! #
-#####################
-
-The spyder and uninstall-spyder commands will only be available in new shell
-sessions. To make them available in this session you must source your .bashrc
-file with:
+These commands will only be available in new shell sessions. To make them
+available in this session, you must source your $shell_init file with:
 
-$ source ~/.bashrc
+$ source $shell_init
 
 ###############################################################################
 
 EOF
 fi
 
 echo "*** Post install script for ${INSTALLER_NAME} complete"
+
+# ----
+[[ -n "$CI" ]] && exit 0  # Running in CI, don't launch Spyder
+if [[ "$OSTYPE" = "darwin"* ]]; then
+    open -a $shortcut_path
+elif [[ "$XDG_CURRENT_DESKTOP" =~ .*(Unity|GNOME|XFCE).* ]]; then
```

Comment:
I think this simply should check if `gtk-launch` is available in PATH and then use it below to launch Spyder. That's because in my case I use KDE and I have `gtk-launch` installed, but due to this check Spyder is not launched automatically for me.

Reply:
Do you have `kde-open`? If so, can you test to see if that works?
```
$ kde-open spyder_spyder
```
Ideally, I'd like to start Spyder using the application shortcut in all circumstances, if possible, including for the `~/.bashrc` alias. Perhaps several fall-back options: `gtk-launch`, then `kde-open` (or desktop environment equivalent), then direct execution (`path/to/spyder`).

Reply:
I have `kde-open5` but that command doesn't launch Spyder (I tried several ways to use but none worked for me). However, running `gtk-launch spyder_spyder` did the trick.

Reply:
Okay, got it. So I will assume that `gtk-launch` is the only reliable desktop application launcher. The script will check for `gtk-launch`, and fall back to direct execution if it is not present.

Reply:
Pushed up changes.

Reply:
> Okay, got it. So I will assume that gtk-launch is the only reliable desktop application launcher.

Yep, it seems that way. I searched for KDE-only alternatives but I couldn't find one.

> The script will check for gtk-launch, and fall back to direct execution if it is not present.

Sounds good to me.